gradle/scripts/dependency.gradle

```diff
@@ -19,6 +19,8 @@ ext.externalDependency = [
     "jsoup"               : "org.jsoup:jsoup:1.8.3",
     "commons_io"          : "commons-io:commons-io:2.5",
     "commons_lang3"       : "org.apache.commons:commons-lang3:3.6",
+    "commons_codec"       : "commons-codec:commons-codec:1.10",
```

Comment:
Is this really needed?

Reply:
Removed now, used it earlier just in case if i had to encode the reader object.